README.md

```diff
@@ -42,6 +44,30 @@ Instead of `$CHIP` you'll need to write the name of your microcontroller.
 For example, one would use `nRF52840_xxAA` for the nRF52840 microcontroller.
 To list all supported chips run `probe-run --list-chips`.
 
+2. Enable debug info
+
+Next check that debug info is enabled for all profiles.
+If you are using the `cortex-m-quickstart` template then this is already the case.
+If not check or add these lines to `Cargo.toml`.
+
+``` toml
+# Cargo.toml
+[profile.dev]
+debug = 1 # default is `true`; not needed if not already overridden
+
+[profile.release]
+debug = 1 # default is `false`; using `true` is also OK
```

Comment:
Why do we need to enable this? Backtraces should work even without line info

Reply:
the virtual unwinder (gimli) requires `.debug_frame` info to compute the stack pointer of the previous frame while walking up the stack. Last time I checked `.debug_frame` is not produced when `debug = false`.

Reply:
Hmm, then how does it work as the backtrace implementation of libstd? That one works even without any debuginfo in the program.

Reply:
Hmm, it seems `.debug_frame` is produced even with `debug = false`. Setting `debug = false` for `profile.dev` makes `probe-run` unwinder fail though; may be due to some other reason.

Reply:
Okay, seems fine to merge this then. Maybe file an issue about the unwinder bug?

Reply:
#9